README.md

```diff
@@ -17,5 +17,3 @@ hello
 ### Description
 
 Will ask the user to enter a name and say "Hello" to them.
-
```

Comment:
I would specifically add here that this program expects a maximum number of 25 characters in the provided name, otherwise an error might occur.